### ISSUE TITLE
Maximized window has top cut off behind panel; top panel wraps to bottom of screen

### ISSUE THREAD

**Issue by jacobgkau** [pop-os/cosmic-comp#447]
After upgrading from 6ba7242 to 9022747 today, whenever I maximize a window, it's displayed with the top cut off behind the top panel. Additionally, the top half of the panel is displayed at the bottom of the screen when this happens.

If I press PrtSc to take a screenshot in this state, it fixes itself before showing the screenshot UI. See below.

https://github.com/pop-os/cosmic-comp/assets/7199422/578a382a-28ca-4498-a859-3151d63293f1

Additionally, if I try to open the Launcher in this state, I've sometimes seen the screen go blank with the top half of the panel repeated seven times. I'm not sure if this is the same issue (I experienced something similar a while back, before the maximize issue started happening). I'm also confused because opening the Launcher previously caused the screen to go back to normal similar to PrtSc did in the video above, before it started making this blinders issue occur.

https://github.com/pop-os/cosmic-comp/assets/7199422/2cf491ac-8d9a-40a2-826f-a28ec336bf53

These videos are of the serw12.

**Comment by WatchMkr** [comment]
I'm unable to reproduce this on the  oryp11. May have been fixed.

**Comment by jacobgkau** [comment]
Yeah, I think this one's been fixed (the second issue is now filed separately at https://github.com/pop-os/cosmic-comp/issues/499).

**Comment by jacobgkau** [comment]
I'm seeing this happen on serw13 again, running 743b5ad (current master).

![1000012131](https://github.com/pop-os/cosmic-comp/assets/7199422/78e56f5c-745a-4c59-ac77-9b6365e58cf1)

**Comment by leviport** [comment]
I think it's fixed again
